schemas/session-replay/full-snapshot-record-schema.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "full-snapshot-record-schema.json",
+    "title": "FullSnapshotRecord",
+    "type": "object",
+    "description": "Schema of a Record type which contains the full snapshot of a screen.",
+    "allOf": [
+        {
+            "$ref": "_common-record-schema.json"
+        },
+        {
+            "required": [
+                "type"
+            ],
+            "properties": {
+                "type": {
+                    "type": "integer",
+                    "description": "The type of this Record.",
+                    "const": 2,
+                    "readOnly": true
+                },
+                "data": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "wireframe-schema.json"
+                    },
+                    "description": "The Wireframes contained by this Record.",
+                    "readOnly": true
+                }
```

Comment:
IMO it's safer to define `data.wireframes: Wireframe[]` (not `data: Wireframe[]`), like proposed in the RFC:
```typescript
data: {
   wireframes: ConcreteWireframe[]
}
```
I mean "safer" as it leaves room for adding other properties to `data` in backward-compatible fashion (without altering the entire type).